sklearn/preprocessing/tests/test_data.py

```diff
@@ -2683,3 +2690,19 @@ def test_power_transformer_constant_feature(standardize):
             assert_allclose(Xt_, np.zeros_like(X))
         else:
             assert_allclose(Xt_, X)
+
+
+@pytest.mark.parametrize(
+    "array_namespace, device, dtype", yield_namespace_device_dtype_combinations()
+)
+def test_standard_scaler_array_api_compliance(array_namespace, device, dtype):
+    xp, device, dtype = _array_api_for_tests(array_namespace, device, dtype)
+
+    from sklearn.datasets import make_classification
+    X, _ = make_classification(random_state=0)
+    X = X.astype(dtype, copy=False)
+
+    X_xp = xp.asarray(X, device=device)
+
+    scale = StandardScaler()
+    X_trans = scale.fit_transform(X_xp)
```

Comment:
The test doesn't do much at the moment.

Reply:
We should reuse the standard checker as done in #26243 and #27110.

Reply:
this should be solved